frontend/src/pages/Token/LaunchForm/Confirm/Ekubo.tsx

```diff
@@ -82,10 +84,19 @@ export default function EkuboLaunch({ previous }: LastFormPageProps) {
 
   // launch
   const launch = useCallback(() => {
-    if (!i129StartingTick || !fees || !chainId || !hodlLimit || !memecoin?.address || !teamAllocationQuoteAmount) return
+    if (
+      !quoteToken ||
+      !i129StartingTick ||
+      !fees ||
+      !chainId ||
+      !hodlLimit ||
+      !memecoin?.address ||
+      !teamAllocationQuoteAmount
+    )
+      return
 
     const uin256TeamAllocationQuoteAmount = uint256.bnToUint256(
-      BigInt(teamAllocationQuoteAmount.multiply(decimalsScale(DECIMALS)).quotient.toString())
+      BigInt(teamAllocationQuoteAmount.multiply(decimalsScale(quoteToken.decimals)).quotient.toString())
```

Comment:
We have to use the memecoin decimals i.e. `DECIMALS`, not the quote token decimals